packages/frontend/src/features/semanticViewer/components/FiltersModal/Filter/MultiStringFilter.tsx

```diff
@@ -54,19 +54,15 @@ const MultiStringFilter: FC<MultiStringFilterProps> = ({
             <Select
                 size="xs"
                 w={75}
+                // paddingRight style on the input field to prevent the text from
+                // being hidden on different OS and browsers
+                styles={{ input: { paddingRight: rem(20) } }}
                 withinPortal
                 data={operatorsOpts ?? []}
                 value={filter.operator}
-                onChange={(value: SemanticLayerFilter['operator'] | null) => {
-                    if (!value) {
-                        return;
-                    }
-
+                onChange={(value: SemanticLayerFilter['operator']) => {
```

Comment:
The default argument for `onChange` in the select component is `string | null`, not sure if removing it won't allow some error to happen

![image](https://github.com/user-attachments/assets/d28f047e-9e05-4843-9f19-bc3a71d4f848)

Reply:
can't be a null because it does not have `clearable`. unfortunately, mantine is not strongly typed that way

Reply:
Gotcha, makes sense 👌 
We can disregard this comment then